index.html

```diff
@@ -6,12 +6,182 @@
+                <div class="options-section__action"><p class="orange options-section__p">Sell</p></div>
+                <div class="options-section__description"><p class="white">Sell your games ang earn prestige as a seller then unlock awesome benefits.</p></div>
+              </div>
+            </li>
+            <li>
+              <div class="options-section__list-content">
+                <div class="options-section__image"><img class="image" src="./images/thirdIcon.png"></div>
+                <div class="options-section__action"><p class="orange options-section__p">Common questions</p></div>
+                <div class="options-section__description"><p class="white">Take time to read the common questions so that you fully understand how the process for selling and 
+                  buying works.</p></div>
+              </div>
+            </li>
+            <li>
+              <div class="options-section__list-content">
+                <div class="options-section__image"><img class="image" src="./images/fourthIcon.png"></div>
+                <div class="options-section__action"><p class="orange options-section__p">Statistics</p></div>
+                <div class="options-section__description"><p class="white">Some people may be skeptikal about our numbers. You can check out our veracity here. You will find our 
+                  statistics on the average seller and buyer.</p></div>
+              </div>
+            </li>
+            <li>
+              <div class="options-section__list-content last-list">
+                <div class="options-section__image"><img class="image" src="./images/fifthIcon.png"></div>
+                <div class="options-section__action"><p class="orange options-section__p">Reviews</p></div>
+                <div class="options-section__description"><p class="white">You can read others reviews to get an idea of the customer satisfaction.</p></div>
+              </div>
+            </li>
+          </ul>
+        </div>
+        <div class="options-section__button-container"><button type="submit" class="options-section__button">Join our community</button></div>
+      </div>
+      <div class="hidden-text__container"><a class="hidden-text">Join our comunity</a></div>
+    </section>
+  </div>
+
+  <!-- Features section -->
+
+  <section class="feature-section">
+    <div class="feature-section__container">
+      <div class="feature-section__title-container">
+        <h1 class="feature-section__title">Popular</h1>
+        <div class="orange-line"></div>
+      </div>
+      <div class="feature-section__games-container">
+      </div>
+      <div class="feature-section__button-container">
+        <button type="submit" class="feature-section__button">
+          <span class="more">LESS</span>
+          <span class="less">MORE</span>
+          <img class="arrow" src="./images/arrow.png">
+          <img class="arrowUp" src="./images/arrowUp.png">
```

Comment:
- [ ] As with all other `img` tags in your project, kindly add an `alt` attribute which will display incase of a broken image link.
- To make your work more professional, kindly tick to this 👍